plugins/modules/make.py

```diff
@@ -90,6 +91,16 @@
     chdir: /home/ubuntu/cool-project
     target: all
     file: /some-project/Makefile
+
+- name: build arm64 kernel on FreeBSD, with 16 parallel jobs
+  community.general.make:
+    chdir: /usr/src
+    jobs: 16
+    target: buildkernel
+    params:
+      -DWITH_FDT:
+      TARGET_ARCH: aarch64
```

Comment:
How about adding comments here:
```suggestion
      # This adds -DWITH_FDT to the command line:
      -DWITH_FDT:
      # This adds TARGET_ARCH=aarch64 to the command line:
      TARGET_ARCH: aarch64
```

Reply:
@felixfontein : I just added the comments with a slight difference( describe 2 variables in a row) , but I think it should be OK.